jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/StartEventTest.java

```diff
@@ -424,9 +425,9 @@ public void afterProcessStarted(ProcessStartedEvent event) {
         });
 
         assertEquals(0, list.size());
-        for (int i = 0; i < 2; i++) {
-            Thread.sleep(1200);
-        }
+        // Timer in the process takes 1000ms, so after 2 seconds, there should be 2 process IDs in the list.
+        // 2100ms is here because it is important to be sure that 2 seconds have passed.
+        Thread.sleep(2100);
```

Comment:
Based on the previous code, shouldn't the wait be 2500ms? Or maybe the 1200ms is wrong and the process indeed takes 1000ms?